packages/conventional-changelog-core/test/test.js

```diff
@@ -65,71 +64,71 @@ betterThanBefore.setups([
     gitDummyCommit(['test9', 'Release note: super release!'])
   },
   function () { // 10
-    shell.exec('git remote add origin https://github.com/user/repo.git')
+    exec('git remote add origin https://github.com/user/repo.git')
   },
   function (context) { // 11
-    shell.exec('git tag -d v0.1.0')
+    exec('git tag -d v0.1.0')
     const tails = gitTails()
     context.tail = tails[tails.length - 1].substring(0, 7)
   },
   function (context) { // 12
-    shell.exec('git tag not-semver')
+    exec('git tag not-semver')
     gitDummyCommit()
 
-    const head = shell.exec('git rev-parse HEAD').stdout.trim()
+    const head = exec('git rev-parse HEAD').trim()
     gitDummyCommit('Revert \\"test9\\" This reverts commit ' + head + '.')
-    context.head = shell.exec('git rev-parse HEAD').stdout.substring(0, 7)
+    context.head = exec('git rev-parse HEAD').substring(0, 7)
   },
   function (context) { // 13
     const tail = context.tail
-    shell.exec('git tag v0.0.1 ' + tail)
+    exec('git tag v0.0.1 ' + tail)
   },
   function () { // 14
     gitDummyCommit()
-    shell.exec('git tag v1.0.0')
+    exec('git tag v1.0.0')
   },
   function () { // 15
     gitDummyCommit()
     gitDummyCommit('something unreleased yet :)')
   },
   function () { // 16
     writeFileSync('./package.json', '{"version": "2.0.0"}') // required by angular preset.
-    shell.exec('git tag foo@1.0.0')
-    mkdirp.sync('./packages/foo')
+    exec('git tag foo@1.0.0')
+    fs.mkdirSync('./packages/foo', { recursive: true })
```

Comment:
`mkdirp` is getting replaced by `mkdirSync` with `recursive` flag

https://nodejs.org/docs/latest-v10.x/api/fs.html#fs_fs_mkdirsync_path_options